gax-java/gax/src/main/java/com/google/api/gax/rpc/ApiCallContext.java

```diff
@@ -66,6 +66,9 @@ public interface ApiCallContext extends RetryingContext {
   /** Returns a new ApiCallContext with the given Endpoint Context. */
   ApiCallContext withEndpointContext(EndpointContext endpointContext);
 
+  /** Returns the EndpointContext stored in the ApiCallContext */
```

Comment:
Why do we need to expose `EndpointContext` here?

Reply:
I think there is a use-case where an user passes in an already created ClientContext: https://github.com/googleapis/sdk-platform-java/blob/c1cd0d1deb74d7aa81f1ad7f6400d193f1464fac/gax-java/gax/src/main/java/com/google/api/gax/rpc/StubSettings.java#L317-L341

I simply use the EndpointContext that was created in ClientContext because there is unfortunately a single param that is resolved inside the ClientContext: the `usingGDCH` param.

Reply:
This has been removed.